### ISSUE TITLE
Add an entry for: Intermittent audio with Google Meet

### ISSUE THREAD

**Issue by janodvarko** [webcompat/knowledge-base#59]
Platform:
https://bugzilla.mozilla.org/show_bug.cgi?id=1670633
https://bugzilla.mozilla.org/show_bug.cgi?id=1666866
https://bugzilla.mozilla.org/show_bug.cgi?id=1653112

Web compat reports:
https://github.com/webcompat/web-bugs/issues/107298
https://github.com/webcompat/web-bugs/issues/107054
https://github.com/webcompat/web-bugs/issues/107098

**Comment by jgraham** [comment]
This is right on the boundary between a perf issue and a compat issue. Functionally it seems like there's a perf issue that makes the site unusable in some cases. I'm not quite sure how much we want to cover such things in the knowledge base.
